src/organisation/OrganisationDetail.vue

```diff
@@ -7,7 +7,7 @@
 
       <div class="row">
         <div class="col">
-          <p class="card-text">Enhedsnr.: {{orgUnit.user_key}}</p>
+          <p class="card-text">Enhedsnr.: {{orgUnit.path}}</p>
```

Comment:
I think we should include this _in addition_ to the user key, rather than _instead of_ it for now. If this customer truly wants to hide the user key, we should do that in configuration. Other customers might consider the user-key/short name useful

Reply:
As I understood Alex, the chosen layout should be used until somebody explicitly wants something else, and then it will need to go into configuration. Ballerup do not want the user_key to be displayed.